packages/web-components/src/components/va-link/va-link.tsx

```diff
@@ -236,6 +242,24 @@ export class VaLink {
       );
     }
 
+    if (external) {
+      return (
+        <Host>
+          <a
+            href={href}
+            rel='noreferrer'
+            class={linkClass}
+            onClick={handleClick}
+            target='_blank'
+          >
+            {text}
+            <va-icon class="external-link-icon" icon="launch"></va-icon>
+            <span class="usa-sr-only">opens in a new window</span>
```

Comment:
Added screen reader only indication that it opens in a new window.

Reply:
This is a good idea. I'm not totally sure if this exact text is right for all situations though.  I was trying to see what the [USWDS guidelines are for external links](https://designsystem.digital.gov/components/link/#link-guidance) and they just say "Provide text context". Does that mean it's the teams should make it clear it's external as part of the link text? Maybe @rsmithadhoc can weight in?

![Screenshot 2024-08-12 at 9 40 40 AM](https://github.com/user-attachments/assets/9a4f4a19-a6a0-42bf-859e-629a5abecf0c)

Reply:
> opens in a new window

@ataker Should we say tab instead of window?

> This is a good idea. I'm not totally sure if this exact text is right for all situations though. I was trying to see what the [USWDS guidelines are for external links](https://designsystem.digital.gov/components/link/#link-guidance) and they just say "Provide text context". Does that mean it's the teams should make it clear it's external as part of the link text? Maybe @rsmithadhoc can weight in?
> 
> ![Screenshot 2024-08-12 at 9 40 40 AM](https://private-user-images.githubusercontent.com/872479/357091008-9a4f4a19-a6a0-42bf-859e-629a5abecf0c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM0NzUwMzIsIm5iZiI6MTcyMzQ3NDczMiwicGF0aCI6Ii84NzI0NzkvMzU3MDkxMDA4LTlhNGY0YTE5LWE2YTAtNDJiZi04NTllLTYyOWE1YWJlY2YwYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgxMlQxNDU4NTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMTlmYzJlY2RlMzIxMDM3NTU5NWVlN2Y5M2E2NDk0MzQwZTc5MGYxNTg0ZWQxNjRjNTZiMzcxMzc0NjUzYTg2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.vXAbvQJ8i5euYjHzq7SOQOkCFWtwQJXpxvvls3DPSh0)

@jamigibbs Yeah, I think that just means provide clear link text that would suggest it goes somewhere else. Instead of `<a href="">go here to find tax return information</a>`, say `<a href="">Here are the IRS guidelines for tax returns</a>`. Or <a href="">Tax Return Guidelines (irs.gov)</a>. So that would be on teams to write that. But we should update the guidance to include information on external links, including this.

Reply:
@rsmithadhoc Is there any overlap or conflict with this text and the `label` prop which adds an aria-label attribute?

Reply:
> @rsmithadhoc Is there any overlap or conflict with this text and the `label` prop which adds an aria-label attribute?

@danbrady `aria-label` replaces the label for assistive technologies, so a screen reader would not read the text label, it would only read the `aria-label` and not the text label. So there shouldn't be any overlap.